src/components/Project/ProjectView.js

```diff
@@ -202,21 +203,30 @@ const ProjectView = React.forwardRef(
               </div>
               <div className="main-info__statistics-section">
                 <ProjectArtifacts
-                  artifacts={projectStore.project.models}
+                  artifacts={groupByUniqName(
+                    projectStore.project.models,
+                    'producer'
```

Comment:
As you can see [here](https://github.com/mlrun/ui/blob/development/src/utils/createArtifactsContent.js#L10), the unique identifier for artifacts is the property `db_key`. Not `produce.name`.
This applies to all artifact categories: Models, Data sets and Files.